### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by logankilpatrick**

### CODE REVIEW

File: examples/Using_embeddings.ipynb
Comment (ARajgor) [inline examples/Using_embeddings.ipynb:1]: You change the client object of openai library. revert to V1. You only need to update the model parameter. 
`client.embeddings.create(input="input", model="text-embedding-3-small").data[0].embedding`

**Comment by aifreak00** [comment]
the embeddings is not working

**Comment by kevinamiri** [comment]
This line is belong to deprecated models, As far as I know, it should new line does not effect of 002, and 003; @logankilpatrick plz ping @ted-at-openai or @arvind-neelakantan 
```
    # replace newlines, which can negatively affect performance.
     text = text.replace("\n", " ")
 ```
 https://github.com/openai/openai-cookbook/blob/c1bd61f944d46e3539b4a136e4273df79b1b7eed/examples/utils/embeddings_utils.py#L20

File: articles/how_to_work_with_large_language_models.md [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
-* how to code
-* etc.
+- how to spell
+- how grammar works
+- how to paraphrase
+- how to answer questions
+- how to hold a conversation
+- how to write in many languages
+- how to code
+- etc.
```
Comment (ttloves2) [inline articles/how_to_work_with_large_language_models.md:16]: 16. how to bring data together
17. speech recognition
18. handwriting recognition
19. summarize articles
20. machine translation

**Review by ttloves2 — COMMENTED** [review]
i added a few ideas, hope i put it in the right place???

**Comment by sankalp1999** [comment]
guys links need to be updated in the docs on here
https://platform.openai.com/docs/guides/embeddings/use-cases#:~:text=Code%20search%20using-,embeddings,-Code_search.ipynb

**Pull request merged**